### ISSUE TITLE
Error 500 on /series endpoint

### ISSUE THREAD

**Issue by brodokk** [brodokk/jellyfin2txt#6]
Error 500 when calling the `/series/` endoint with the following error:

```
Traceback (most recent call last):
  File "/home/root_man/jellyfin2txt/.venv/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/root_man/jellyfin2txt/.venv/lib/python3.9/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/root_man/jellyfin2txt/.venv/lib/python3.9/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/root_man/jellyfin2txt/.venv/lib/python3.9/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/root_man/jellyfin2txt/app.py", line 170, in series
    return get_series()
  File "/home/root_man/jellyfin2txt/app.py", line 78, in get_series
    series_ids = get_series_ids()
  File "/home/root_man/jellyfin2txt/app.py", line 49, in get_series_ids
    return get_media_ids(params)
  File "/home/root_man/jellyfin2txt/app.py", line 34, in get_media_ids
    for item in media['Items']:
TypeError: 'NoneType' object is not subscriptable
```

**Comment by brodokk** [comment]
This error is due because the SERIES_ID was incorrect. Sadly the jellyfin python module don't seems to raise correctly an error when a problem come.

I will see to have a better handling of errors, until then the issue will still be open.

**Comment by brodokk** [comment]
Fixed with commit c3f08e41da7c579bfd4f78092638969b31abfd42
